src/mflux_debugger/debugger_service.py

```
@@ -0,0 +1,1933 @@
+            message_parts.append("🧹 Cleared Python cache for next debug session.")
+
+            message = "\n".join(message_parts)
+
+            return DebuggerResponse(success=True, message=message)
+        except Exception as e:  # noqa: BLE001
+            return DebuggerResponse(success=False, message=f"❌ Error: {e}", error=str(e))
+
+    def _clear_python_cache(self):
+        """Clear Python __pycache__ directories and .pyc files."""
+        try:
+            import subprocess  # noqa: PLC0415
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)